### PR TITLE
Revised comparison relational operator articles

### PR DESCRIPTION
**Pull request by pkulikov**
Supported by dotnet/samples#523

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening to start a new build.

File: docs/csharp/language-reference/operators/greater-than-operator.md [hunk 1]
```diff
@@ -11,17 +10,25 @@ helpviewer_keywords:
...
-- [C# Programming Guide](../../../csharp/programming-guide/index.md)  
-- [C# Operators](../../../csharp/language-reference/operators/index.md)  
-- [explicit](../../../csharp/language-reference/keywords/explicit.md)
+
+The "greater than" relational operator `>` returns `true` if its first operand is greater than its second operand, `false` otherwise. All numeric and enumeration types support the `>` operator. For operands of the same [enum](../keywords/enum.md) type, the corresponding values of the underlying integral type are compared.
+
+The following example demonstrates the usage of the `>` operator:
+
+[!code-csharp-interactive[greater than example](~/samples/snippets/csharp/language-reference/operators/GreaterAndLessOperatorsExamples.cs#Greater)]
+
```
Comment (BillWagner) [inline docs/csharp/language-reference/operators/greater-than-operator.md]: This comment applies to all articles.  Following the example, I'd add the following note and link:

> Any relational comparison where one of the values is `NaN` (not a number) returns `false.`  The `NaN` value is neither greater than, less than, nor equal to any other `double` value.  For more information and examples, see the <xref:System.Double.NaN> reference article.

**Review by BillWagner — CHANGES_REQUESTED** [review]
These are great changes @pkulikov 

I have one suggestion, based on feedback from developers on the ongoing confusion regarding NaN and comparisons.

File: docs/csharp/language-reference/operators/greater-than-operator.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/operators/greater-than-operator.md]: @BillWagner thank you for sharing the developer feedback. I've added the note; it's slightly different from your suggestion. Please check the latest commit and let me know if the text should be updated.

Also, I've added examples of comparison with the `NaN` value: dotnet/samples#525

Comment (BillWagner) [reply docs/csharp/language-reference/operators/greater-than-operator.md]: That's perfect @pkulikov 

I'll approve this and :shipit: now.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**